sdk/src/ssi/did_test.go

```diff
@@ -1,15 +1,13 @@
 package ssi
 
 import (
-	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestGenerateDIDKey(t *testing.T) {
-	didWrapper, err := GenerateDIDKey("RSA")
+	resultBytes, err := GenerateDIDKey("RSA")
 	assert.NoError(t, err)
-	assert.True(t, strings.HasPrefix(didWrapper.DIDKey, "did:key:"))
-	assert.NotEmpty(t, didWrapper.PrivateJSONWebKey)
+	assert.NotEmpty(t, resultBytes)
```

Comment:
I think it makes sense to keep. It also acts as documentation on how to use. 
```suggestion
	var result map[string]any
	assert.NoError(json.Unmarshal(resultBytes, &result))
	assert.True(t, strings.HasPrefix(result["didKey"].(string), "did:key:"))
	assert.NotEmpty(t, result["privateJwk"])
```

Reply:
Oh awesome. I was wondering how to marshal this into a map for testing, but couldn't figure it out. Updated!